prow/cmd/horologium/main.go

```diff
@@ -46,7 +46,8 @@ func main() {
 	}
 
 	// start a cron
-	cr := cron.New()
+	cronLogger := logrus.WithField("agent", "cron")
```

Comment:
nit: This field should be added in the constructor for `Cron`.

Reply:
awww right :-( I was referencing somewhere that passes the entry along

Reply:
You can still make it accept a `logrus.Entry`, but the "agent" field should probably be added in the constructor.

Reply:
"agent" conflicts with pj.Spec.Agent. Should this be just `client`?

Reply:
That makes sense to me. If "agent" is used to log `pj.Spec.Agent` then this should be changed: https://github.com/kubernetes/test-infra/pull/5442/files#diff-0e5c8bf65ff28507861c63b6bbe7274aR88

Reply:
done

Reply:
We still don't log that field but it's possible to do it in the future.